website/docs/content/writing-guidelines/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Writing guidelines
+caption: Consistent language guidelines for components and patterns.
+description: Consistent language guidelines for components and patterns.
+previewImage: assets/illustrations/content/writing-guidelines.jpg
+---
+
+## Date and time
+
+Display dates and times clearly and consistently across products.
+
+### Usage
+
+These formatting options are available in the [Time](#) component. Choose the format most appropriate for your user needs.
+
+- **Relative date**: Displays time in relative terms, e.g., "3 hours ago" or "in 2 days". This format is recommended for recent events, particularly within the past or upcoming week, as it simplifies recognition of time-sensitive actions.
+- **Friendly date**: Provides a user-friendly date and time, e.g., "Sep 5, 2018, 3:30 PM EST". Ideal for displaying the date and time to who, or what, set it. Limit time format to the minute.
```

Comment:
Suggestion: it may just be me, but I am not really sure what "Ideal for displaying the date and time to who, or what, set it" means. what does it mean to display a time to what set it?

Reply:
I agree that it's confusing.

Reply:
Added a suggested simplification below.

(HashiConf isn't a person and it doesn't set a time or read a time.)